nft_ingester/src/program_transformers/bubblegum/mod.rs

```diff
@@ -32,56 +32,76 @@ where
     T: ConnectionTrait + TransactionTrait,
 {
     let ix_type = &parsing_result.instruction;
-    match ix_type {
+
+    // @TODO this would be much better served by implemneting Debug trait on the InstructionName
```

Comment:
you already can remove `debug!` from the match here and use it after
`debug!("{ix_str} instruction:)`

Reply:
Leaving the info! only